### DISCUSSION TITLE
What is the status of this project?

### DISCUSSION THREAD

**Discussion by MisinformedDNA** [microsoft/winget-cli#709] [Q&A] [answered]
[v0.3](https://github.com/microsoft/winget-cli/milestone/5) is now 4 months behind. There have only been 10 commits in the last 30 days. **10!** Does MS have a strong enough interest in this? Maybe they should drop this and instead contribute to an established open source system like `AppGet` or `Chocolately`?

**Comment by doctordns** [comment]
A better way to put this might be to ask the team to update their milestones?

**Comment by MisinformedDNA** [comment]
They actually [updated them](https://github.com/microsoft/winget-cli/commit/6f7fb63d94ef2d56ff310efaca98b1c1949ae41e) a few weeks ago, but they didn't change any of the dates or give any other timelines.

**Reply by doctordns** [reply]
I meant update their milestone dates! :-)

**Comment by denelon** [comment]
@MisinformedDNA We're balancing back end and front end work. I was using the dates on the actual milestones more for sequencing than anything else. Is there something specific you're looking for?

**Reply by MisinformedDNA** [reply]
@denelon I just haven't been able to get the preview version through the store (might be due to org settings?)

UPDATE:

I finally created a Win10 VM to test it. It worked fine. So the fault must be with the org. I guess I could just do the manual installation...

**Comment by ri-we** [comment]
I'm looking into using winget for our developers. But I'm worried about the timelines, seams to be little progress. Should we invest in winget?

**Reply by denelon** [reply]
@ri-we the Windows Package Manager still in preview. We just released "import" as a stable feature today, and we published the .admx and .adml files to support Group Policy management via ADMX Ingestion. What features do your developers need? Several commands like "list", "upgrade", "uninstall", and "export" are currently experimental so they can be used. It just depends on whether the product meets your needs.

**Reply by ri-we** [reply]
I would like my developers to be able to install, update and uninstall software themselves preferably not as local admin and from a trusted source. Also I would be nice if an Admin could determine which software they can install.

**Reply by denelon** [reply]
I think the only gap to call out is Admin oversight on individual packages. For that scenario, the organization can have a private repository of trusted applications. We're working on the REST API currently. Once that is released, the client can be restricted to certain "source" repositories by policy.

**Comment by ghost** [comment]
@denelon 

It seems winget won't be able to reach v1.0 in May  when it still lacks 
Dependency Support, Multiple Architectures, Multiple Languages, and User vs. System installation, Third party REST source, Metered Networks, .zip, and .exe support.  

shouldn't  milestones be updated now?

**Reply by denelon** [reply]
I've updated all of the milestones and simplified them down to release style milestones. I've also updated the roadmap doc to adjust to align more with what we're planning on delivering for 1.0.

**Reply by ghost** [reply]
its amazing seeing all the work being done here. thanks 👍 I really appreciate it.

**Comment by denelon** [comment, marked as answer]
Just as an FYI, after 1.0 we're going to focus on quality for the features released in 1.0. There are plenty of challenges with list, upgrade, and export. We're looking at the data on installer success rates and package success rates. I think the major items we're considering for 1.1 are dependencies, PowerShell cmdlets, .zip and .exe. There are also a couple of smaller features like submitting hash validation failures and reporting on other issues with packages via a "report" command. Be sure to add your 👍 on Issues you see of high value.

We were ambitious with our goals for 1.0 and needed to scale back on features to ensure we deliver high quality. We will use the v.Next milestone to help narrow down our prioritized features for inclusions in future releases like v1.1 and v1.2. At this point we are looking essentially at quarterly releases.

**Reply by ghost** [reply]
Quarterly releases is a beautiful idea. Unlike per year basis  releases,  It will surely give engineers a reality check time to time and will also help them to focus on particular issues.   I see no reason for not doing it.

**Reply by ghost** [reply]
@denelon 
 quarterly releases should have been the way from the get go. just like how project reunion has started to release. please do it .
